frontend/src/components/MapView/LeftPanel/ChartsPanel/index.tsx

```diff
@@ -514,43 +580,91 @@ const ChartsPanel = memo(
       ) : null;
       return [locationBox, ...titles, ...zipped];
     }, [
-      admin0Key,
-      adminLevel,
       adminProperties,
-      classes.textLabel,
+      startDate1,
```

Comment:
is there a reason for the re-order?

Reply:
I am pretty sure it was done automatically. I am reordering them.